_benchmark/user-guide/understanding-results.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: default
+title: Understanding results
+nav_order: 22
+parent: User guide
+---
+
+
+At the end of each test run, a summary table is produced which includes metrics like service time, throughput, latency and more. These metrics provide insights into how the workload selected performed on a benchmarked OpenSearch cluster.
+
+The following guide gives information about to understand the results of the summary report and what steps to take before running another test on your cluster.
```

Comment:
about how to understand the summary results

Reply:
I think we can also remove "what steps to take before running another test on your cluster" as that's not found here.